resources/multisite_drupal_standard.make

```diff
@@ -653,6 +653,15 @@ projects[scheduler][version] = 1.5
 
 projects[scheduler_workbench][subdir] = "contrib"
 projects[scheduler_workbench][version] = 1.3
+; Allow to schedule the publish date of a revision
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-1999
+; https://www.drupal.org/project/scheduler_workbench/issues/2048999
+projects[scheduler_workbench][patch][] = https://www.drupal.org/files/issues/scheduler_workbench-revision_publish-2048999-45.patch
+; Allow publication of revisions with selected status. This patch requires the patch from issue https://www.drupal.org/project/scheduler_workbench/issues/2048999
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-1999
+; This patch requieres the patch https://www.drupal.org/files/issues/scheduler_workbench-revision_publish-2048999-45.patch from
+; this issue https://www.drupal.org/project/scheduler_workbench/issues/2048999.
```

Comment:
We need a reference to the D.O issue from where this "local" patch id derived like that we can ensure the traceability of the patch and ease our job during future module upgrades.

Reply:
Comment updated